hw/ip/i2c/doc/registers.md

```diff
@@ -492,30 +492,33 @@ I2C target address and mask pairs
 I2C target acquired data
 - Offset: `0x58`
 - Reset default: `0x0`
-- Reset mask: `0x3ff`
+- Reset mask: `0x7ff`
 
 ### Fields
 
 ```wavejson
-{"reg": [{"name": "ABYTE", "bits": 8, "attr": ["ro"], "rotate": 0}, {"name": "SIGNAL", "bits": 2, "attr": ["ro"], "rotate": -90}, {"bits": 22}], "config": {"lanes": 1, "fontsize": 10, "vspace": 80}}
+{"reg": [{"name": "ABYTE", "bits": 8, "attr": ["ro"], "rotate": 0}, {"name": "SIGNAL", "bits": 3, "attr": ["ro"], "rotate": -90}, {"bits": 21}], "config": {"lanes": 1, "fontsize": 10, "vspace": 80}}
 ```
 
 |  Bits  |  Type  |  Reset  | Name                       |
 |:------:|:------:|:-------:|:---------------------------|
-| 31:10  |        |         | Reserved                   |
-|  9:8   |   ro   |    x    | [SIGNAL](#acqdata--signal) |
+| 31:11  |        |         | Reserved                   |
+|  10:8  |   ro   |    x    | [SIGNAL](#acqdata--signal) |
 |  7:0   |   ro   |    x    | [ABYTE](#acqdata--abyte)   |
 
 ### ACQDATA . SIGNAL
 Host issued a START before transmitting ABYTE, a STOP or a RESTART after the preceeding ABYTE
 
-| Value   | Name    | Description                                         |
-|:--------|:--------|:----------------------------------------------------|
-| 0x0     | NONE    | ABYTE contains ordinary data byte as received       |
-| 0x1     | START   | ABYTE contains the 8-bit I2C address (R/W in lsb)   |
-| 0x2     | STOP    | ABYTE contains junk                                 |
-| 0x3     | RESTART | ABYTE contains junk, START with address will follow |
+| Value   | Name      | Description                                                                                                                                              |
+|:--------|:----------|:---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 0x0     | NONE      | ABYTE contains ordinary data byte as received                                                                                                            |
+| 0x1     | START     | ABYTE contains the 8-bit I2C address (R/W in lsb)                                                                                                        |
+| 0x2     | STOP      | ABYTE contains junk                                                                                                                                      |
+| 0x3     | RESTART   | ABYTE contains junk, START with address will follow                                                                                                      |
+| 0x4     | NACK      | ABYTE contains either the address or data that was NACK'ed                                                                                               |
+| 0x5     | NACKSTART | ABYTE contains the I2C address which was ACK'ed, but the block will continue and NACK the next data byte that was received: this only happens for writes |
```

Comment:
Can this commit be squashed into the previous commit (with the commit messages merged), please?

Reply:
Yes :smile: